plugin/src/sbt-test/psv/custom_ivy_home_config/src/main/scala/Demo.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2019-2024 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+package com.here.bom
+
+import com.fasterxml.jackson.databind.ObjectMapper
+
+object Demo extends App {
```

Comment:
Do we run this app somewhere or mention it?

Reply:
Actually in this test we do no run this app. But we compile it

Reply:
I'd name it like, `CustomIvyTest` instead of `Demo`

Reply:
Done